contrib/go.mongodb.org/mongo-driver/mongo/mongo.go

```diff
@@ -48,6 +48,8 @@ func (m *monitor) Started(ctx context.Context, evt *event.CommandStartedEvent) {
 		tracer.Tag(ext.DBType, "mongo"),
 		tracer.Tag(ext.PeerHostname, hostname),
 		tracer.Tag(ext.PeerPort, port),
+		tracer.Tag(ext.Component, "mongo-go-driver"),
+		tracer.Tag(ext.SpanKind, "client"),
```

Comment:
Span kind is limited to a few values as well: server, client, producer, consumer, internal. Maybe it's worth adding corresponding internal constants, in case we ever have to go back to this issue

Reply:
Added